web/src/sidebar_ui.js

```diff
@@ -28,6 +28,14 @@ export function hide_streamlist_sidebar() {
     $(".app-main .column-left").removeClass("expanded");
 }
 
+export function any_sidebar_expanded() {
+    if (window.innerWidth > media_breakpoints_num.xl) {
+        // Sidebars are always visible beyond xl breakpoint.
+        return false;
+    }
+    return Boolean($("[class^='column-'].expanded").length);
+}
```

Comment:
It would be nice to move this earlier in the branch, since it's one of the lower risk changes. I'm about to merge 4 of the early commits, and this one looks good aside from the merge conflict. But I guess we need to decide the naming questions first.

Reply:
Done.